content/en/news/releases/1.22.x/announcing-1.22/change-notes/index.md

```diff
@@ -0,0 +1,246 @@
+  ([Issue #48634](https://github.com/istio/istio/issues/48634))
+
+## istioctl
+
+- **Updated** `istioctl experimental proxy-stauts` has been promoted to `istioctl proxy-status`. The old `istioctl proxy-status` command has been removed.
+This promotion should not result in any loss of functionality. However, the request is now sent based on xDS instead of HTTP, and we have introduced a set of new xDS-based flags to target the control plane.
+
+- **Added** support for multi-cluster analysis in `istioctl analyze` command when there are remote cluster secrets set up through [Install Multicluster](/docs/setup/install/multicluster/).
+
+- **Added** a new `istioctl dashboard proxy` command, which can be used to show the admin UI of different proxy pods, like Envoy, Ztunnel, Waypoint.
+
+- **Added** `--proxy` option to `istioctl experimental wait` command.
+  ([Issue #48696](https://github.com/istio/istio/issues/48696))
+
+- **Added** namespace filtering to `istioctl pc workload` using the `--workloads-namespace` flag, to display workloads in a specific namespace.
+
+- **Added** `istioctl dashboard istio-debug` to display the Istio debug endpoints dashboard.
+
+- **Added** `istioctl x describe` supports displaying the details of policies for `PortLevelSettings`.
+  ([Issue #49802](https://github.com/istio/istio/issues/49802))
+
+- **Added** ability to define the traffic address type (service, workload, all or none) for waypoints via the `--for` flag when using the `istioctl experimental waypoint apply` command.  ([Issue #49896](https://github.com/istio/istio/issues/49896))
+
+- **Added** Allow user to name their waypoint through istioctl via --name flag on the waypoint command.
+**Removed** Remove ability for user to specify service account for the waypoint by deleting the --service-account flag on the waypoint command
+  ([Issue #49915](https://github.com/istio/istio/issues/49915)),([Issue #50173](https://github.com/istio/istio/issues/50173))
+
+- **Added** Allow user to enroll their waypoint in the waypoint's namespace through istioctl via --enroll-namespace flag on the waypoint command.  ([Issue #50248](https://github.com/istio/istio/issues/50248))
+
+- **Added** Add ztunnel-config istioctl command. Allow users to view Ztunnel configuration information through istioctl via ztunnel-config workload flag.
+**Removed** Remove workload flag from proxy-config command. Users must use ztunnel-config command to view Ztunnel configuration information.  ([Issue #49841](https://github.com/istio/istio/issues/49841))
+
+- **Added** Warn user in standard out when `istioctl experimental waypoint apply --enroll-namespace` is called and namespace is not labeled ambient.  ([Issue #50396](https://github.com/istio/istio/issues/50396))
+
+- **Added** Add --for flag to istioctl x waypoint generate command so that the user can preview the yaml before they apply it.  ([Issue #50790](https://github.com/istio/istio/issues/50790))
```

Comment:
```suggestion
- **Added** the `--for` flag to `istioctl experimental waypoint generate` command so that the user can preview the YAML before they apply it.
  ([Issue #50790](https://github.com/istio/istio/issues/50790))
```